openc3-cosmos-script-runner-api/app/models/running_script.rb

```diff
@@ -343,18 +302,42 @@ def self.spawn(scope, name, suite_runner = nil, disconnect = false, environment
     model = OpenC3::OfflineAccessModel.get_model(name: username, scope: scope) if username and username != ''
```

Comment:
Don't need the `if username` check anymore because you're always setting it to `Anonymous` above if it's not set

Reply:
Will change to check for not Anonymous.